### PR TITLE
Fix small code errors

### PR DESCRIPTION
**Pull request by berlinum**

### CODE REVIEW

File: src/components/PasswordInput.js [hunk 1]
```diff
@@ -7,25 +7,25 @@ const Input = styled.input`
...
-  };
+  }
 
   return (
     <Input
       type="password"
-      value={value}
-      onChange={event => setValue(event.target.value)}
-      color={color}
+      inputValue={value}
```
Comment (lmachens) [inline src/components/PasswordInput.js]: This doesn't seem right. `Input` doesn't know `inputValue`.

File: src/components/PasswordInput.js [hunk 2]
```diff
@@ -7,25 +7,25 @@ const Input = styled.input`
...
+      bgColor={color}
```
Comment (lmachens) [inline src/components/PasswordInput.js]: The semantic usage of props is for information about the structure and content of a component, not about the styling. I do not recommend to have props like `color`, `fontSize`, `margin`, ...

File: src/components/PasswordInput.js (hunk 1 above)
Comment (berlinum) [reply src/components/PasswordInput.js]: true! 🙈

File: src/components/PasswordInput.js (hunk 2 above)
Comment (berlinum) [reply src/components/PasswordInput.js]: Changed it, tnx!

**Pull request merged**